packages/cw-orch-core/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 pub mod contract;
 pub mod env;
+pub use env as CwOrchEnvVars;
```

Comment:
oeeehh yikes! Have you seen this pattern/naming somewhere before? I don't really like it. People will think it's a type when it isn't.

Reply:
Yeah you're right, will change